src/core/linalg/src/sparse/4C_linalg_sparsematrixbase.hpp

```diff
@@ -138,9 +138,9 @@ namespace Core::LinAlg
     {
       if (!row_map_)
       {  // check if view is uninitialized
-        row_map_ = Core::LinAlg::Map(sysmat_->RowMap());
+        row_map_.emplace(sysmat_->RowMap());
       }
-      return *row_map_;
+      return row_map_->sync(sysmat_->RowMap());
```

Comment:
@knarfnitram @maxfirmbach This call should ensure that we update the view if necessary. Minimal tests passed, let's see what the checks on Github say. I need to add more docs and cleanup, but I think this might work :pray: